LibTest/js_interop/NullRejectionException/nullRejectionException_t02.dart

```diff
@@ -27,7 +27,9 @@ main() {
     Expect.fail("NullRejectionException expected");
   }).onError((e, st) {
     Expect.isTrue(e is NullRejectionException);
-    Expect.isTrue((e as NullRejectionException).isUndefined);
+    if (!isWasm) {
```

Comment:
Do we have a `isJs`? That would make this easier to read without the negation: `if (isJs) { ... }`.

Reply:
Replaced with `isJS`